packages/flutter/lib/src/material/two_level_list.dart

```diff
@@ -14,19 +14,35 @@ import 'list_tile.dart';
 import 'theme.dart';
 import 'theme_data.dart';
 
+/// This enum is deprecated. Please use [ListTileTheme] instead.
+enum MaterialListType {
+  /// A list tile that contains a single line of text.
+  oneLine,
+
+  /// A list tile that contains a [CircleAvatar] followed by a single line of text.
+  oneLineWithAvatar,
+
+  /// A list tile that contains two lines of text.
+  twoLine,
+
+  /// A list tile that contains three lines of text.
+  threeLine,
+}
+
+/// This constant is deprecated. The [ListTile] class sizes itself based on
+/// its content and [ListTileTheme].
+@deprecated
+Map<MaterialListType, double> kListTileExtent = const <MaterialListType, double>{
+  MaterialListType.oneLine: 48.0,
+  MaterialListType.oneLineWithAvatar: 56.0,
+  MaterialListType.twoLine: 72.0,
+  MaterialListType.threeLine: 88.0,
+};
+
 const Duration _kExpand = const Duration(milliseconds: 200);
 
-/// An item in a [TwoLevelList] or a [TwoLevelSublist].
-///
-/// A two-level list item is similar to a [ListTile], but a two-level list item
-/// automatically sizes itself to fit properly within its ancestor
-/// [TwoLevelList].
-///
-/// See also:
-///
-///  * [TwoLevelList]
-///  * [TwoLevelSublist]
-///  * [ListTile]
+/// This class is deprecated. Please use [ListTile] instead.
+@deprecated
```

Comment:
That didn't seem to what the `@Deprecated` parameter was for 
- https://api.dartlang.org/stable/1.23.0/dart-core/Deprecated-class.html @Deprecated()